includes/application/forwardApplication.php

```diff
@@ -63,26 +63,55 @@
 	
 	if( needApprover($status) )
 	{
-		echo <<<HTML
+		if( isset($_SESSION['PostName']) )
+		{
+			echo <<<HTML
 <form action="forwardApprover.php" method="post">
 <input type="text" name="app_id" value=$app_id style="visibility: hidden; display: none;" readonly>
 <input type="text" name="app_type" value=$app_type style="visibility: hidden; display: none;" readonly>
 <input type="submit" value="Forward to Next Person"><br><br><br>
 </form>
 HTML;
+		}
+		else
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after ELSE keyword; newline found

<!-- Comment by [SideCI](https://sideci.com) -->